just/libafl.just

```diff
@@ -0,0 +1,41 @@
+# Main Justfile for LibAFL
+# Provides multiple useful variables.
+#
+# Must be set:
+#   - `FUZZER_NAME`: Name of the executable.
+#
+# Provides:
+#   - `PROFILE`: Profile (either `dev` or `release`). Default is `release`.
+#   - `PROFILE_DIR`: Profile directory (either `debug` or `release`).
+#   - `TARGET_DIR`: target directry. Defaults to `target`.
+#   - `BUILD_DIR`: Root directory in which the program is compiled.
+#   - `FUZZER`: Executable path.
+
+PROFILE := env("PROFILE", "release")
+PROFILE_DIR := if PROFILE == "dev" { "debug" } else { "release" }
+TARGET_DIR := absolute_path(env("TARGET_DIR", "target"))
+BUILD_DIR := TARGET_DIR / PROFILE_DIR
+FUZZER_EXTENSION := if os_family() == "windows" { ".exe" } else { "" }
+FUZZER := BUILD_DIR / FUZZER_NAME + FUZZER_EXTENSION
```

Comment:
on windows it's "\" instead of "/" so i think you don't need to add these `FUZZER_EXTENSION` because it won't work anyway on cmd.exe

Reply:
on powershell it works
on cmd.exe no

and we are using cmd.exe